src/run/run.c

```diff
@@ -1021,20 +1024,35 @@ static void run_context_check_done(RunContext *c) {
                 sd_event_exit(c->event, EXIT_SUCCESS);
 }
 
+static int map_job(sd_bus *bus, const char *member, sd_bus_message *m, sd_bus_error *error, void *userdata) {
+        bool *b = userdata;
+        const char *job;
+        uint32_t id;
+        int r;
+
+        r = sd_bus_message_read(m, "(uo)", &id, &job);
+        if (r < 0)
+                return r;
+
+        *b = id != 0 || !streq(job, "/");
```

Comment:
It's unclear to me why both of those variables need to be checked. E.g. in systemctl `print_property`, "0" is not printed. Wouldn't be enough to do the same here, and ignore the second item?

Reply:
yes, either id is 0 and the job is "/"  or id is != 0 and job != "/". But it will not happen that matches and the other doesn't also...

I wasn't sure which one was the best to check, so i checked both to not having to make a choice ;-)